lighthouse_runner.js

```diff
@@ -100,17 +103,28 @@ const processResults = (processObj) => {
     } else {
         filePath = path.join(tempFilePath, replacedUrl + '.mobile.report.' + opts.output);
     }
+    // // @TODO ASYNC IS BETTER BUT WE ARENT CURRENTLY AWAITING THE PROMISE PROPERLY
```

Comment:
With the original version, I don't think the promise was being properly `await`ed so I was getting errors when my new code tried to interact with the _last_ individual report. I haven't had time to study it closely, but I think there was a race condition that just didn't matter until I added more moving peaces down-stream. I could be wrong, but switching to the synchronous version of `writeFile` fixed the problem immediately.